.github/workflows/pr-lint.yml

```diff
@@ -7,10 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check ticket name conforms to requirements
-        run: 'echo ${{ github.event.pull_request.title }} | grep -i -E -q "^((apm|niad)-[0-9]+): ([a-zA-Z]+)|(dependabot\/)"'
+        run: 'echo ${{ github.event.pull_request.title }} | grep -i -E -q "^((niad)-[0-9]+): ([a-z]+)|(dependabot\/)"'
```

Comment:
Are these "(" or ")" actually needed now? Also, doesn't your regex just accept one word? Cannot we have spaces between the words?

Reply:
It will accept more than one word after the JIRA key. See if you can work out why!

Reply:
Oh, because we don't end with $. True